pallets/migration/src/lib.rs

```diff
@@ -351,5 +388,34 @@ pub mod pallet {
 				.into(),
 			)
 		}
+
+		/// This extrinsic disables the call-filter. After this has been called the chain will accept
+		/// all calls again.
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::finalize())]
+		#[transactional]
+		pub fn finalize(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin)?;
+
+			ensure!(
+				<Completed<T>>::get() == false,
+				Error::<T>::MigrationAlreadyCompleted
+			);
+
+			<Completed<T>>::put(true);
+
+			Self::deposit_event(Event::<T>::MigrationFinished);
+
+			Ok(())
+		}
+	}
+}
+
+impl<T: Config> Contains<<T as frame_system::Config>::Call> for Pallet<T> {
+	fn contains(c: &<T as frame_system::Config>::Call) -> bool {
+		if !pallet::Completed::<T>::get() {
```

Comment:
why not just add this filter to the BaseFilter ?
I'm not sure I fully understand.

Reply:
Wdym?

Reply:
This pallet is the filter of `system::config`. Just want to have the ability to remove the filter via an extrinisic.

Reply:
so this would not allow any calls including the one's we have enabled when the migration is underway ?

Reply:
nvm. Should work

Reply:
This allows every call we provide the migration pallet filter type with. I pass our normal `BaseFilter` here. So its the same filter as we normally use during the start off the chain on Altair for example. Once we are through with this, we can call the `finalize` extrinsic and afterward all calls are enabled.